requirements.txt

```diff
@@ -1,6 +1,11 @@
 importlib-metadata==4.11.1
 pytest==7.0.1
+pytest-asyncio
 pytest-cov==3.0.0
 sentinel==0.3.0
+# Will sqlalchemy 2.0 work with create_engine(future=True) in unittets?
 sqlalchemy>=1.4
 strawberry-graphql>=0.95
+testing.postgresql>=1.3
```

Comment:
At my company we run it locally with no problem, and isn't too heavy weight (test setup takes on the order of 1s)! Just need to have postgres installed on the device, which I'll add to our devcontainer once we have a devcontainer. testing.postgresql is basically just a convenient wrapper around postgres to make it easy (and performant) to use in tests. We also then use github actions to run a testing docker container, which of course has postgres installed.

Speaking of github actions, it appears none ran for this PR, so my money is on this library not having any CI/CD setup yet. It looks like strawberry core uses devcontainer and github actions, which is awesome because my company does the same so I'm familiar with both. In the next PR, I'm going to try to just copy over .devcontainer and .github from strawberiy-grpahql/strawberry, some config files from the root directory, make a few tweaks, and see if that'll get us CI/CD for ~free